draft-irtf-cfrg-opaque.md

```diff
@@ -507,17 +515,19 @@ The server output of this stage is a record corresponding to the client's
 registration that it stores in a credential file alongside other client
 registrations as needed.
 
-The registration flow is shown below:
+The registration flow, shown below, uses three messsage structures named
+`RegistrationRequest`, `RegistrationResponse`, and `RegistrationRecord`. Their
+contents and wire format are defined in {{registration-messages}}.
 
 ~~~
     creds                                   parameters
       |                                         |
       v                                         v
     Client                                    Server
     ------------------------------------------------
-                registration request
+                      request
```

Comment:
```suggestion
                      RegistrationRequest
```

Reply:
I changed this twice, I still didn't understand what is the convention.

What must be on top of the arrow? the object type or an instance of such an object. I can change either way, but want to know what is the convention to follow.

Reply:
I don't know if we have a convention, but using the type here seems most clear.